### PR TITLE
Changed global sdk version in global.json

### PR DESCRIPTION
**Pull request by DistroRickDev**
Changed global sdk version in global.json
Format changed to MAJOR.MINOR only

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "6.0.0",
+    "version": "6.1",
```
Comment (DistroRickDev) [inline global.json]: Please make sure the sdk version is 6.0 you changed to 6.1

**Review by DistroRickDev — COMMENTED** [review]
All good only one comment regarding wrong minor version

File: global.json (hunk 1 above)
Comment (DistroRickDev) [reply global.json]: Changed accordingly

**Review by LordB0red — APPROVED** [review]
All good, 6.0 is what works for all

**Pull request merged**